py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -23,9 +23,12 @@ class Batch:
 
     def __init__(self, config_type: ConfigType) -> None:
         self.config_type = config_type
-        self.filenames: List[str] = []
         self.total_size = 0
 
+        self.prefix = ""
+        self.suffix = ""
+        self.filenames: List[str] = []
```

Comment:
store all of the compressed stuff so that we don't have to run the compression algo to check against event string size every time we add a file.